test/9.js

```diff
@@ -0,0 +1,12 @@
+const solution = require('../solutions/9');
+
+const test = (num1, num2, result) => {
+  if (solution(num1, num2) === result) {
+    console.log(`Correct!`);
+  } else {
+    console.log(`Wrong. Expected: ${result}. Actual: ${solution(num1, num2)}`);
+  }
+};
+
+test(3, 6, 3);
```

Comment:
can you test this: `test(15, 6)`

Reply:
It came out as wrong but I fixed it. Thanks.